packages/ui/src/components/text-field.css

```
@@ -125,4 +132,46 @@
       letter-spacing: var(--letter-spacing-normal);
     }
   }
+
+  /* Ghost variant: borderless, only bottom border, used in search bars / inline editors */
+  &[data-variant="ghost"] {
+    [data-slot="input-wrapper"] {
+      display: flex;
+      align-items: center;
+      width: 100%;
+
+      padding: 10px 12px;
```

Stylelint (17.10.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)